pymica_tests/pymica2_test.py

```diff
@@ -380,18 +380,28 @@ def test_init_interpolate_id2d(self):
                            'variables_files': {
                                'altitude': 'pymica_tests/data/tifs/altitude.tif',
                                'd_coast': 'pymica_tests/data/tifs/d_coast.tif'}}}
-
+        
         with open('pymica_tests/data/config_test.json', 'w') as f:
             json.dump(config, f)
             f.close()
 
+        
+        data_dict = [{'id': 'C6', 'value': 6.4, 'altitude': 264,
+                      'lon': 0.95172, 'lat': 41.6566,
+                      'dist': 0.8583929293407604},
+                     {'id': 'C7', 'value': 5.6, 'altitude': 427,
+                      'lon': 1.16234, 'lat': 41.66695,
+                      'dist': 0.8387222708681318}]
+        
+        
+
         id2d = PyMica('id2d', 'pymica_tests/data/config_test.json')
-        field = id2d.interpolate('pymica_tests/data/init_data.json')
+        field = id2d.interpolate(data_dict)
 
```

Comment:
Why the result values have changed? The methodology is the same, the result, I think, it should be the same.

Reply:
I think that the code is still ok, but the data used to interpolate should be loaded from the same json file, and then pass just de dictionary you get.

Reply:
Solved